### ISSUE TITLE
Incorrect request from WCS Team Engine Test Case for DescribeCoverage

### ISSUE THREAD

**Issue by hemant559** [opengeospatial/ets-wcs10#30]
Team Engiine Test Suite (1.13),
There is wrong a URL for the test case(wcs1-0-0:getcoverage_operations-getcoverage_request-grid_size-post-xml-2).

Here POST-XML request for DescribeCoverage is not proper.
Request d1e10485_1:
```
   Method: post
   URL: http://122.15.218.129:8081/OGC_WCS/OGCReqResDispatcher?
   Body: 

               <DescribeCoverage service="WCS" version="1.0.0">
                 <Coverage>sikkim</Coverage>
               </DescribeCoverage>
```
Ideal request which gets validated by most servers - 

```
<DescribeCoverage
  version="1.0.0"
  service="WCS"
  xmlns="http://www.opengis.net/wcs"
  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
  xsi:schemaLocation="http://www.opengis.net/wcs http://schemas.opengis.net/wcs/1.0.0/describeCoverage.xsd">
    <Coverage>sikkim</Coverage>   
</DescribeCoverage>

```

**Comment by dstenger** [comment]
Thank you for reporting. We will do further investigation.

**Comment by lgoltz** [comment]
Fixed with #33. schemaLocation was not added as not necessary for a valid request. Tested with http://122.15.218.129:8081/OGC_WCS/OGCReqResDispatcher?SERVICE=WCS&REQUEST=GetCapabilities, now the same error as described in #31 occurs.
